src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -217,26 +226,49 @@ class IOUAmountPage extends React.Component {
-                        placeholder={this.props.numberFormat(0)}
-                        keyboardType={CONST.KEYBOARD_TYPE.NUMBER_PAD}
-                    />
+                    {isCurrencySymbolToLeft ? (
+                        <>
+                            <TouchableOpacity onPress={() => Navigation.navigate(this.props.hasMultipleParticipants
+                                ? ROUTES.getIouBillCurrencyRoute(this.props.reportID)
+                                : ROUTES.getIouRequestCurrencyRoute(this.props.reportID))}
+                            >
+                                <Text style={styles.iouAmountText}>{currencySymbol}</Text>
+                            </TouchableOpacity>
+                            <TextInput
+                                disableKeyboard
+                                autoGrow
+                                hideFocusedState
+                                inputStyle={[styles.iouAmountTextInput, styles.p0, styles.noLeftBorderRadius, styles.noRightBorderRadius]}
+                                textInputContainerStyles={[styles.borderNone, styles.noLeftBorderRadius, styles.noRightBorderRadius]}
+                                onChangeText={this.updateAmount}
+                                ref={el => this.textInput = el}
+                                value={formattedAmount}
+                                placeholder={this.props.numberFormat(0)}
+                                keyboardType={CONST.KEYBOARD_TYPE.NUMBER_PAD}
+                            />
+                        </>
+                    ) : (
+                        <>
+                            <TextInput
+                                disableKeyboard
+                                autoGrow
+                                hideFocusedState
+                                inputStyle={[styles.iouAmountTextInput, styles.p0, styles.noLeftBorderRadius, styles.noRightBorderRadius]}
+                                textInputContainerStyles={[styles.borderNone, styles.noLeftBorderRadius, styles.noRightBorderRadius]}
+                                onChangeText={this.updateAmount}
+                                ref={el => this.textInput = el}
+                                value={formattedAmount}
+                                placeholder={this.props.numberFormat(0)}
+                                keyboardType={CONST.KEYBOARD_TYPE.NUMBER_PAD}
+                            />
+                            <TouchableOpacity onPress={() => Navigation.navigate(this.props.hasMultipleParticipants
+                                ? ROUTES.getIouBillCurrencyRoute(this.props.reportID)
+                                : ROUTES.getIouRequestCurrencyRoute(this.props.reportID))}
+                            >
+                                <Text style={styles.iouAmountText}>{currencySymbol}</Text>
+                            </TouchableOpacity>
+                        </>
+                    )}
```

Comment:
99% code of this ternary is duplicate.

Reply:
is this the best way to set UI Elements direction?

Reply:
Is this a better approach?

```javascript

        const currencySymbolElement = (
            <TouchableOpacity onPress={() => Navigation.navigate(this.props.hasMultipleParticipants
                ? ROUTES.getIouBillCurrencyRoute(this.props.reportID)
                : ROUTES.getIouRequestCurrencyRoute(this.props.reportID))}
            >
                <Text style={styles.iouAmountText}>{currencySymbol}</Text>
            </TouchableOpacity>
        );

        const amountTextInput = (
            <TextInput
                disableKeyboard
                autoGrow
                hideFocusedState
                inputStyle={[styles.iouAmountTextInput, styles.p0, styles.noLeftBorderRadius, styles.noRightBorderRadius]}
                textInputContainerStyles={[styles.borderNone, styles.noLeftBorderRadius, styles.noRightBorderRadius]}
                onChangeText={this.updateAmount}
                ref={el => this.textInput = el}
                value={formattedAmount}
                placeholder={this.props.numberFormat(0)}
                keyboardType={CONST.KEYBOARD_TYPE.NUMBER_PAD}
            />
        );

       {isCurrencySymbolLTR ? [currencySymbolElement, amountTextInput] : [amountTextInput, currencySymbolElement]}

```

Reply:
Flex-direction row-reverse should be enough.

Reply:
That works but that is not good in terms of accessibility, pressing tab will go to first element in order it exist in dom.

Reply:
Can we use the TabIndex prop for this? Otherwise, using variables looks good.

Reply:
> Warning: Avoid using tabindex values greater than 0. Doing so makes it difficult for people who rely on assistive technology to navigate and operate page content. Instead, write the document with the elements in a logical sequence.

Source: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex